zerver/views/documentation.py

```diff
@@ -144,17 +158,11 @@ def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
         context["article"] = documentation_article.article_path
         not_index_page = not context["article"].endswith("/index.md")
 
-        if documentation_article.article_path.startswith("/") and os.path.exists(
-            documentation_article.article_path
-        ):
+        if os.path.exists(documentation_article.article_path):
```

Comment:
Why is the `startswith` removed here? The comment just below says this is the absolute path case.

One thing we should be careful about in this code path is security; we want to make sure you can't visit a URL like zulip.example.com/help/../../../../etc/zulip/zulip-secrets.conf; which I'm pretty sure is enforced mainly through the `urls.py` configuration.

Reply:
I added the `startswith` check back in.

The `urls.py` configuration for api and policies views are both `slug:article`, so that is enforced there.

Your example above for help views happens in `get_path`, when we set article to be "missing" for any case other than "include/sidebar_index" ...
```python
        elif article == "include/sidebar_index":
            pass
        elif "/" in article:
            article = "missing"
            http_status = 404
```